src/cli/output.rs

```diff
@@ -35,7 +35,7 @@ impl Output<'_> {
         let text_position_x: usize = checkerboard_size + 2 * config.padding;
         let text_position_y: usize = 0;
 
-        let mut canvas = Canvas::new(checkerboard_size, 51, config.brush);
+        let mut canvas = Canvas::new(checkerboard_size, 60, config.brush);
```

Comment:
Why is this necessary?

Reply:
Transparent colors will use the extended `rgba()`/`hsla()` notation, which takes an additional 8 characters for alpha values with 3 digit precision:

```
# no transparency ---

  ▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀  RGB: rgb(255, 215, 0)
  ▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀  HSL: hsl(51, 100.0%, 50.0%)
--------------------------------------------------|
                                                  51 chars

# with transparency ---

  ▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀  RGB: rgba(255, 215, 0, 0.123)
  ▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀  HSL: hsla(51, 100.0%, 50.0%, 0.123)
------------------------------------------------------|
                                                      55 chars
```

I think it could be reduced from `60` to `55`?

Reply:
Ah, I see! To be honest, I don't remember where the 51 came from. I believe it was a worst-case bound (long color names?), not a rough guess. But I'm not sure anymore.